.github/workflows/wagon-tests.yml

```diff
@@ -64,6 +64,55 @@ jobs:
         run: |
           bundle exec rake app:rubocop
 
+  wagon_seeds:
+    name: seeds
+    runs-on: 'ubuntu-20.04'
+    env:
+      HEADLESS: true
+      RAILS_DB_ADAPTER: postgresql
+      RAILS_DB_HOST: 127.0.0.1
+      RAILS_DB_PORT: 5432
+      RAILS_DB_USERNAME: hitobito
+      RAILS_DB_PASSWORD: hitobito
+      RAILS_DB_NAME: hitobito_test
+      RAILS_TEST_DB_NAME: hitobito_test
+      RAILS_ENV: test
```

Comment:
Maybe we should run the seeds in the `development` environment?

Reply:
yes, I would say so.

Reply:
Fixed.